fdbserver/DataDistribution.actor.h

```diff
@@ -50,10 +50,13 @@ enum {
 	PRIORITY_TEAM_HEALTHY    = 140,
 	PRIORITY_TEAM_CONTAINS_UNDESIRED_SERVER = 150,
 
-	PRIORITY_MERGE_SHARD     = 240,
-	PRIORITY_SPLIT_SHARD     = 250,
+	// Set removing_redundant_team priority lower than merge/split_shard_priority,
+	// so that removing redundant teams does not block merge/split shards.
+	PRIORITY_TEAM_REDUNDANT  = 200,
```

Comment:
I think this change will have impact on Status reporting, i.e., 
https://github.com/apple/foundationdb/blob/c3a8ae47527f5291b1265ae4a4e4113ed759eb50/fdbserver/Status.actor.cpp#L1302, since these "else if" is in descending order.

Also, https://github.com/apple/foundationdb/blob/c3a8ae47527f5291b1265ae4a4e4113ed759eb50/fdbserver/Status.actor.cpp#L1258 might need some thought too. 

What's the priority inversion you mentioned in the commit message?

Reply:
The priority band is calculated by the priority / 100. 
Each priority band has some amount of budget to move data in parallel. 
If we put redundant team priority into the same band with merge/split shard priority, removing redundant team can take up the budget and stops the merge/split shard from running. That's how priority inversion happens.

Reply:
Yes, you are right. 
The new commit fix the problems.